Mixpanel/MixpanelPeople.m

```diff
@@ -109,9 +109,7 @@ - (void)addPeopleRecordToQueueWithAction:(NSString *)action andProperties:(NSDic
     }
 }
 
-#pragma mark - Public API
-
-- (void)addPushDeviceToken:(NSData *)deviceToken
+- (void)pushDeviceTokenToString:(NSData *)deviceToken
```

Comment:
@yarneo Does implementing the method like this work? Let me know if I can update this further!